.travis.yml

```diff
@@ -2,7 +2,7 @@ language: python
 python:
   - "3.6"
 before_install:
-  - nvm install 6.12.2
+  - nvm install
```

Comment:
Lets see if we can get Travis to reuse `.nvmrc` instead of hard coding a version